src/test/resources/TestNG.xml

```diff
@@ -6,9 +6,15 @@
 
     <test name="Example Test" preserve-order="false">
         <classes>
+<<<<<<< Updated upstream
             <class name="Homework17" />
 <!--            <class name="ProfileTests"/>-->
 <!--            <class name="BaseTest"/>-->
+=======
+            <class name="LoginTests"/>
+            <class name="BaseTest"/>
+            <class name="Homework16"/>
+>>>>>>> Stashed changes
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 


![image](https://user-images.githubusercontent.com/104761482/212854770-fb60c810-2187-4ed0-9f18-516f8e45207e.png)

for example 

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG">
    <parameter name="BaseURL" value="https://bbb.testpro.io/" />

    <test name="Example Test" preserve-order="false">
        <classes>
        <class name="Homework16"/>
        </classes>
    </test>
</suite>

```

Reply:
Sorry, I don't know how to correct it. Any idea what should be done? I've edited my TestNG.xml file.